README.md

```diff
@@ -41,10 +41,7 @@ def project do
 
       # ...
     ],
-    application: [
```

Comment:
In Elixir 1.4+ you don't need to specify it, due to application inference. In fact specifying the app will likely break other deps (unless you list them all in `applications`).